gossipd/gossipd.c

```diff
@@ -1538,6 +1538,10 @@ static struct io_plan *get_incoming_channels(struct io_conn *conn,
 			const struct half_chan *hc;
 			struct route_info *ri;
 
+			/* Don't leak private channels. */
+			if (!is_chan_public(c))
```

Comment:
This will need to change if we ever decide to not announce channels ourselves.

Reply:
Good thought. Not sure what the right thing is here though: why did we want it private?